python/dgl/nn/pytorch/gt/egt.py

```diff
@@ -0,0 +1,178 @@
+"""EGT Layer"""
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class EGTLayer(nn.Module):
+    r"""EGTLayer for Edge-augmented Graph Transformer (EGT), as introduced in
+    `Global Self-Attention as a Replacement for Graph Convolution
+    Reference `<https://arxiv.org/pdf/2108.03348.pdf>`_
+
+    Parameters
+    ----------
+    ndim : int
+        Node embedding dimension.
+    edim : int
+        Edge embedding dimension.
+    num_heads : int
+        Number of attention heads, by which :attr: `ndim` is divisible.
+    num_vns : int
+        Number of virtual nodes.
+    dropout : float, optional
+        Dropout probability. Default: 0.0.
+    attn_dropout : float, optional
+        Attention dropout probability. Default: 0.0.
+    activation : callable activation layer, optional
+        Activation function. Default: nn.ELU().
+    ffn_multiplier : float, optional
```

Comment:
ffn_hidden_size_multiplier? Shall we elaborate more on how it affects the hidden size of feed forward layers for node and edge update?

Reply:
Since `ffn_multiplier` is set to 1.0 for all configs, I'd like to remove it later.

Reply:
ok, fair enough.